<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Recuperar Senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailInput">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="NomeInput">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="EmailLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Informe o email:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NomeLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NovaSenhaInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
